index.php

```diff
@@ -1 +1,24 @@
 <?php
+
+$db = new PDO (
+    'mysql:host=192.168.20.20; dbname=Collector',
+    'root',
+    ''
+);
+
+$db->setAttribute(
+    PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC
+);
+
+$sql = $db->prepare('SELECT * FROM `garden_plants`');
```

Comment:
I would specify all the field names here, you don't know how a database will change over time, someone might add 5,000 new fields and that will slow your app down by miles